HIPS/hip-1.md

```diff
@@ -0,0 +1,54 @@
+---
+hip: <to be assigned>
+title: <HIP title>
+author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s), e.g. (use with the parentheses or triangular brackets): FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
+status: Example
+type: <Standard Track | Informational>
+category (*only required for Standard Track): <Core | Interface | Economics | HRC>
+created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+requires (*optional): <HIP number(s)>
+replaces (*optional): <HIP number(s)>
+---
+
+Hedera Improvement Proposal (HIP) Template
+
+## Simple Summary
+Please provide a simple summary of your improvement proposal. 
+
+It's often beneficial to assume that those reading this are not extremely familiar with Hedera development.
+
+## Abstract
+This section is best suited for a short length technical description of the issues being addressed.
+
+## Motivation
+Please elaborate in a few sentences why this would be an improvement to the existing ecosystem of Hedera software. 
+
+This could be a problem it solves, a use case being developed that could benefit, etc.
+
+## Specification
+The technical specification should describe the syntax and semantics of any new feature. 
```

Comment:
Should mention please include any images that help visualize the design like UML diagrams, flowcharts, etc. These should be named with the same prefix as the proposal and checked into the same directory.